mflowgen/Tile_MemCore/construct.py

```diff
@@ -114,6 +120,8 @@ def construct():
   synth.extend_inputs( ['sram_tt.lib', 'sram.lef'] )
   #pt_signoff.extend_inputs( ['sram_tt.db'] )
   genlibdb.extend_inputs( ['sram_tt.lib'] )
+  if multicorner:
+      genlibdb_ff.extend_inputs( ['sram_tt.lib'] )
```

Comment:
Wrong one? tt?

Reply:
yep. I'll fix now.

Reply:
Seconded, seems it should be `sram_ff.lib`